### DISCUSSION TITLE
Issue with HintAssist.Foreground Not Working

### DISCUSSION THREAD

**Discussion by WWWWaiting** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3716] [Q&A]
I’m trying to use HintAssist.Foreground to change the hint text color, but it doesn’t seem to have any effect. Has anyone else experienced this issue or know how to properly set the hint foreground color?
![image](https://github.com/user-attachments/assets/203e34d7-73f5-4ba7-a40e-477b4c700193)

**Comment by nicolaihenriksen** [comment]
@WWWWaiting The `HintAssist.Foreground` brush is only applied when the hint is in its floating position. I don't believe you currently have a good way of changing it away from its default brush in the resting position.

If you start typing text in the `TextBox`, you should see the `Red` brush come into effect on the hint as it floats.

**Reply by sminys13** [reply]
Hello. Do I understand correctly that everything related to Hint.Assist works only when an element is selected?
For example, materialDesign:HintAssist.HintOpacity does not change the intensity of the hint, even when the element is selected.
<img width="402" height="91" alt="image" src="https://github.com/user-attachments/assets/309210f1-d123-47a1-9303-668025f41240" />
only the intensity of the combobox triangle changes
<img width="400" height="91" alt="image" src="https://github.com/user-attachments/assets/80126cdb-5d00-441e-8115-375be9a8d97f" />
<img width="400" height="87" alt="image" src="https://github.com/user-attachments/assets/b3cd8fa9-5ffc-48d9-82b1-88dd22acc35a" />

<img width="446" height="42" alt="image" src="https://github.com/user-attachments/assets/da73a150-402c-4755-bad2-966d2058841c" />

This applies to version 5.2.1.
